Dockerfile

```diff
@@ -1,16 +1,13 @@
-FROM kong:0.11
+FROM kong:2.0.3
```

Comment:
Kong maintainers released version 2.0.4, which was theoretically fixed as a memory leak compared to version 2.0.3. Perhaps it would be interesting to update to this newer version. The CHANGELOG is not large, so there is likely to be no impact on the work that has already been done.
https://github.com/Kong/kong/blob/master/CHANGELOG.md#204